compiler/src/iree/compiler/Dialect/Flow/Transforms/FuseDequantizationMatmul.cpp

```diff
@@ -10,12 +10,17 @@
 #include "mlir/Dialect/Arith/IR/Arith.h"
 #include "mlir/Dialect/Linalg/IR/Linalg.h"
 #include "mlir/Dialect/Linalg/Transforms/Transforms.h"
+#include "mlir/Dialect/Math/IR/Math.h"
 #include "mlir/Dialect/Tensor/Utils/Utils.h"
 #include "mlir/IR/PatternMatch.h"
 #include "mlir/Pass/Pass.h"
 #include "mlir/Support/LLVM.h"
 #include "mlir/Transforms/GreedyPatternRewriteDriver.h"
 
+#define DEBUG_TYPE "iree-flow-fuse-dequantization-matmul"
+#define DBGS() (llvm::dbgs() << "[" DEBUG_TYPE "]: ")
+#define LDBG(X) LLVM_DEBUG(DBGS() << X << "\n")
```

Comment:
Prefer not creating new schemes for individual files - it makes it harder to move code across files/projects and harder to read for people as they need to learn pass-specific macros. A few extra characters that easier to maintain and understand for everyone are worth it.